modules/deployment-model-impl/src/main/java/org/mule/runtime/module/deployment/impl/internal/policy/DefaultApplicationPolicyInstance.java

```diff
@@ -214,4 +219,14 @@ public Optional<Policy> getOperationPolicy() {
         .map(chain -> new Policy(chain, parametrization.getId()));
   }
 
+
+  /**
+   * Check if ApiGateway sent
```

Comment:
Is this JDoc sentence truncated?
Or maybe you meant to say to check this only if the policy was provided by ApiGateway? but if that's the case, how would the caller know?

I think we can avoid unnecessarily referencing to ApiGateway if this is a parameter that could be present in any policy (even if not documented).